test/Elastic.Apm.Tests/ConfigTest.cs

```diff
@@ -335,21 +335,24 @@ public void SetMetricsInterValTo1M()
 		public void SetMetricsInterValTo10()
 			=> MetricsInterValTestCommon("10").Should().Be(10 * 1000);
 
+		/// <summary>
+		/// Any negative value should be treated as 0
+		/// </summary>
 		[Fact]
 		public void SetMetricsInterValToNegative5()
-			=> MetricsInterValTestCommon("-5").Should().Be(ConfigConsts.DefaultValues.MetricsIntervalInMilliseconds);
+			=> MetricsInterValTestCommon("-1").Should().Be(0);
 
 		[Fact]
 		public void SetMetricsInterValToNegative5S()
```

Comment:
nit: same as above: method name should be adjusted.

Reply:
Done